src/main/java/mServer/crawler/sender/MediathekHr.java

```diff
@@ -19,26 +19,35 @@
  */
 package mServer.crawler.sender;
 
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Date;
-
-import de.mediathekview.mlib.Config;
 import de.mediathekview.mlib.Const;
 import de.mediathekview.mlib.daten.DatenFilm;
-import de.mediathekview.mlib.tool.Functions;
+import de.mediathekview.mlib.daten.ListeFilme;
 import de.mediathekview.mlib.tool.Log;
-import de.mediathekview.mlib.tool.MSStringBuilder;
-import mServer.crawler.CrawlerTool;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
 import mServer.crawler.FilmeSuchen;
-import mServer.crawler.GetUrl;
+import mServer.crawler.sender.hr.HrSendungOverviewCallable;
+import mServer.crawler.sender.hr.HrSendungenDto;
+import mServer.crawler.sender.hr.HrSendungenListDeserializer;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
 
 public class MediathekHr extends MediathekReader {
 
     public final static String SENDERNAME = Const.HR;
-    private MSStringBuilder seite = new MSStringBuilder(Const.STRING_BUFFER_START_BUFFER);
-    private MSStringBuilder rubrikSeite = new MSStringBuilder(Const.STRING_BUFFER_START_BUFFER);
-
+    
+    private final static String URL_SENDUNGEN = "http://www.hr-fernsehen.de/sendungen-a-z/index.html";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)